App.TestModule/App.Example.csproj

```diff
@@ -5,11 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="4.4.0" />
```

Comment:
Why you have changed this module ?

Reply:
I just manage nuget packages

Reply:
And connect this , because I have it in button restore nuget packages